src/Targets/Microsoft.VisualBasic.DesignTime.targets

```diff
@@ -31,21 +31,32 @@
       </PropertyPageSchema>
 
      <ProjectCapability Include="VB;Managed"/>
+  </ItemGroup>
+
+  <!-- Targets -->
+
+  <!-- Returns Vbc command-line arguments for the language service -->
+  <Target Name="CompileDesignTime"
+          Returns="@(_CompilerCommandLineArgs)"
+          DependsOnTargets="_CheckCompileDesignTimePrerequisite;Compile"
+          Condition="'$(IsCrossTargetingBuild)' != 'true'"
+        >
+
+    <!-- VB Framework projects needs core library references. In the case of Command line scenario, the vbc injects this reference. -->
+    <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
```

Comment:
This would mean that PCLs are now broken they have a different TFI - what about xamarin? This is a rabbit hole you don't want to go down. Just replicating the same logic as the old LS and the compiler is the easiest and most consistent.